src/composables/queries/usePoolQuery.ts

```diff
@@ -82,7 +82,6 @@ export default function usePoolQuery(
     await injectTokens([
       ...tokensListExclBpt(pool),
       ...tokenTreeLeafs(pool.tokens),
-      pool.address,
```

Comment:
I went back in history and injecting the pool address has been in the code forever but I couldn't figure out why we'd need it, as most pools don't have a price so it's making it do unnecessary calls to coingecko and slowing things done.